keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2706,7 +2565,7 @@ calls happens inside the E2E test.
 
 All tests that don't involve actually running a Pod can become part of
 conformance testing. Those tests that run Pods cannot be because CDI support in
-runtimes is not required.
+runtimes and plugin support in the kubelet are not required for conformance.
```

Comment:
For reference: [`#sig-node` discussion around standardizing plugin support](https://kubernetes.slack.com/archives/C0BP8PW9G/p1744031821315639?thread_ts=1743781404.041849&cid=C0BP8PW9G)